2/contrib/s2i/run

```diff
@@ -131,17 +131,15 @@ if [ "${CONTAINER_MEMORY_IN_BYTES}" -lt "${DEFAULT_MEMORY_CEILING}" ]; then
 
     JAVA_MAX_HEAP_PARAM="-Xmx${CONTAINER_HEAP_MAX}m"
     if [ -z $CONTAINER_INITIAL_PERCENT ]; then
-      # jboss default was 100% or ms==mx
-      JAVA_INITIAL_HEAP_PARAM="-Xms${CONTAINER_HEAP_MAX}m"
-    else
-      CONTAINER_INITIAL_HEAP=$(echo "${CONTAINER_HEAP_MAX} ${CONTAINER_INITIAL_PERCENT}" | awk '{ printf "%d", $1 * $2 }')
-      JAVA_INITIAL_HEAP_PARAM="-Xms${CONTAINER_INITIAL_HEAP}m"
+      CONTAINER_INITIAL_PERCENT=0.07
     fi
+    CONTAINER_INITIAL_HEAP=$(echo "${CONTAINER_HEAP_MAX} ${CONTAINER_INITIAL_PERCENT}" | awk '{ printf "%d", $1 * $2 }')
+    JAVA_INITIAL_HEAP_PARAM="-Xms${CONTAINER_INITIAL_HEAP}m"
 fi
 
 if [ -z "$JAVA_GC_OPTS" ]; then
   # note - MaxPermSize no longer valid with v8 of the jdk ... used to have -XX:MaxPermSize=100m
-  JAVA_GC_OPTS="-XX:+UseParallelGC -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:MaxMetaspaceSize=100m"
+  JAVA_GC_OPTS="-XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:MaxMetaspaceSize=100m"
```

Comment:
is the metaspace one of the things that's runed by the cgroup limits setting?  can we get rid of that argument here?  it's been a source of pain as noted by @gabemontero and others.

Reply:
I think it makes sense to rely on the cgroup setting to size metaspace as appropriate and let the JVM do appropriate expansions if needed. By explicitly setting the upper bound on the metaspace it's more likely to run into the limit.

+1 on removing it.

Reply:
So we are agreed, nuke it as part of this PR please @jerboaa 

When doing so, if you don't mind, add a comment with something like `we no longer set metaspace becausethe JVM should expand metaspace until it reaches the container limit. See http://hg.openjdk.java.net/jdk8u/jdk8u/hotspot/file/4dd24f4ca140/src/share/vm/memory/metaspace.cpp#l1470 `